internal/resources/resources.go

```diff
@@ -33,6 +33,7 @@ func Apply(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec, containerNam
 
 	resources.Requests[corev1.ResourceCPU] = *resource.NewQuantity(res.CPU, resource.DecimalSI)
 	resources.Requests[corev1.ResourceMemory] = *resource.NewQuantity(res.Memory*gigabyte, resource.BinarySI)
+	resources.Requests[corev1.ResourceEphemeralStorage] = *resource.NewQuantity(res.Disk*gigabyte, resource.BinarySI)
```

Comment:
Should we also add a default disk resource request when .resources is not defined? See `if res == nil` conditional above.

Reply:
Yes great catch. I should probably add some tests